etc/generated_registry_auth.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- type: dockerhub
+  user: dymurray
+  pass: qci-test
```

Comment:
Probably want to ignore the generated file

Reply:
Oops, this shouldn't be there.